blockchain/utxobackend_test.go

```diff
@@ -225,27 +225,72 @@ func TestPutUtxos(t *testing.T) {
 				backendEntries[outpoint] = entry
 			}
 		}
+		if err != nil {
```

Comment:
I don't see any reason to check this here again.  This is probably a rebase error that snuck in and perhaps belongs in a future commit I haven't reviewed yet.

For reference, this is in the commit for `blockchain: Add UtxoBackend interface.`.

Reply:
Good catch, correct that this stuck into a later commit during a rebase.